src/app/backend/resource/generic/controller.go

```diff
@@ -47,6 +56,8 @@ type ResourceController interface {
 	UID() types.UID
 	// Get is a method, that returns ResourceOwner object.
 	Get(allPods []v1.Pod, allEvents []v1.Event) ResourceOwner
+	// Returns all log sources of controlled resource (e.g. a list of containers and pods for a replica set)
+	GetLogSources(k8sClient *client.Clientset) LogSources
```

Comment:
I like the idea of adding more code here. Ideally we would use interfaces for all resources and our code would be more consistent.